tests/base-notebook/test_packages.py

```diff
@@ -69,6 +69,7 @@
     "ca-certificates",
     "conda-forge::blas[build=openblas]",
     "hdf5",
+    "jupyter_server[version='>=2.0.0']",  # Temporary fix for: https://github.com/jupyter/docker-stacks/issues/1851
```

Comment:
It looks like this only test if the import of this package works or not. Try to remove `[version='>=2.0.0']`

Reply:
No, this won't work, because our parsing is a bit broken.
I fixed the test (not in the beautiful way, but it works).